WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+- **Components:** V, S, M (a shard of glass or ice)
+- **Duration:** 1 minute
+___
+Your fingers become cold to the touch as frost manifests on their surface. Make a melee spell attack against a creature within range. On a hit, the creature is incapacitated and restrained until the spell ends as ice encap&shy;sulates their body. A target who hits the creature with an attack breaks the ice and ends the spell.
+
+#### Frostfire Bolt
+*1st-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 120 feet
+- **Components:** V, S, M (a piece of glass and sulfur)
+- **Duration:**  Instantaneous
+___
+You hurl a bolt of blue flames at a creature within range. Make a ranged spell attack against the creature. On a hit, the target takes 2d6 fire damage and 2d6 cold damage.
+
+***At Higher Levels.*** When you cast this spell using a spell slot of 2nd level or higher, the fire and cold damage increases by 1d6 for each slot level above 1st.
+
+\columnbreak
+
+#### Glacial Spike
+*7th-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 120 feet
+- **Components:** V, S
+- **Duration:** Instantaneous
+___
+You form a large spike of ice above your head, and launch it at a creature that you can see within range, causing it to shatter across their body. The target must make a Dexterity saving throw. It takes 8d8 + 40 cold damage on a failed save, or half as much damage on a successful one. 
+
+If this damage reduces a creature to 0 hit points, it is reduced to shards of ice, leaving behind everything it was wearing and carrying. The creature can be restored to life only by means of a true resurrection or a wish spell.
+
+#### Ice Block
+*4th-level abjuration*
+___
+- **Casting Time:** 1 reaction, when you take damage
+- **Range:** Self
+- **Components:** V, S
+- **Duration:**  Concentration, up to 1 minute
+___
+Ice rapidly materializes around you, protecting you from harm. You gain 40 temporary hit points and immunity to cold damage for the spells duration. While the spell is active you can't move and take any actions or reactions. The spell remains until the temporary hit points have been expend or until your concentration breaks.
+
+#### Ice Nova
+*8th-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 120 feet
+- **Components:** V, S, M (glass dust and water)
+- **Duration:** Instantaneous
+___
+Frigid cold rapidly expands in a 15-foot-radius centered on a point you you can see within range. Each creature in the cold must succeed on a Dexterity saving throw. On a failed save, a creature takes 10d6 cold damage and is restrained for 1 minute as ice encases the lower part of their legs. On a successful save, it takes half as much damage and isn't restrained by this spell.
+
+A creature restrained by the ice can use its action to make a Strength check against your spell save DC. On a success, it frees itself. Alternatively can the ice be broken by dealing a total of 40 hit points worth of damage to it, its armor class is 10. 
```

Comment:
What about, "Alternatively, the ice can be broken"?

Also probably a . before "Its armor class is 10".

Reply:
Alternatively, the ice can be broken by dealing a total of 40 hit points worth of damage to it. It has an armor class of 10.

Reply:
Sounds good! 👍